src/components/cylc/tree/TreeItem.vue

```diff
@@ -91,6 +91,12 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
               />
             </div>
             <span class="mx-1">{{ node.name }}</span>
+            <span
+              v-if="!isExpanded && latestJob(node)?.platform"
+              class="mx-1 text-grey"
+            >
+              [{{ latestJob(node)?.platform }}]
```

Comment:
Almost ready, thanks! Just one more thing: don't need brackets here
```suggestion
              {{ latestJob(node)?.platform }}
```